compiler/parser.mly

```diff
@@ -48,9 +48,12 @@
 %token VOID_LITERAL
 
 /* Precedence and associativity of each operator */
+%left EQ NEQ
+%left LCAR LEQ RCAR GEQ
```

Comment:
I think EQ and NEQ have to have lower precedence, consider
```
true == 3 < 2
````
or
```
a < b == c < d
```

I pulled the precedence out of the C operator precedence table so it should be the same!